codeexamples/SYCLBufferExampleExecutingCUDAKernels/StaticLibraryMakeCUDAKernel/CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+
+## [main]
+if(NOT DEFINED ${CMAKE_BUILD_TYPE})
+	set(CMAKE_BUILD_TYPE "Debug")
```

Comment:
The CMakeLists higher up the tree may not define/reset CMAKE_BUILD at all and nor may it be specified on the command line. This ensures the CMAKE_BUILD is defined and set to a known value. I see this similar to the good practice of setting C++ variables with an initial know value and not a random hope/assumed it will be always be 0 - sometimes it may be some it may not --> boom.